gen_def.sh.in

```diff
@@ -0,0 +1,44 @@
+#! /bin/sh
+
+# overriding user localization settings for sort
+export LC_ALL=C
+
+HEADER='LIBRARY wpcap.dll\nEXPORTS'
+TMPFILE=DELETEME.c
+DEF=Win32/Prj/wpcap.def
+
+# The order matters. pcap/export-defs.h must come first
+cat pcap/export-defs.h pcap/pcap.h pcap/namedb.h nametoaddr.c > $TMPFILE
+
+# The order doesn't matter. Just don't include anything and make sure that the names
+# of all PCAP_API declarations come directly after the second space from the left.
+sed -i -e "s/#include <.*>//g" \
```

Comment:
-i is a non-standard extension; it's not listed in [the Single UNIX Specification page for sed](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/sed.html).

You should probably have sed read the files directly, without a cat command; the SUS page for sed *does* say it supports multiple file name arguments, and "If multiple file operands are specified, the named files shall be read in the order specified and the concatenation shall be edited."  Redirect the sed command's standard output to $TMPFILE.